File: samples/c/01table/table.c
#include <stdio.h>

#define LIMIT 5


int main(int argc, char** argv) {
    printf("   ");
    for (int i = 1; i <= LIMIT; i++) {
        printf("%-3d", i);
    }
    printf("\n");
    for (int a = 1; a <= LIMIT; a++) {
        printf("%-3d", a);
        for (int b = 1; b <= LIMIT; b++) {
            printf("%-3d", a*b);
        }
        printf("\n");
    }
}

File: samples/php/01table/table.php
<?php

define("LIMIT", 5);


for ($a = 0; $a <= LIMIT; $a++) {
    if ($a == 0) {
        printf("%-3s", "");
    } else {
        printf("%-3d", $a);
    }
    for ($b = 1; $b <= LIMIT; $b++) {
        if ($a == 0) {
            printf("%-3d", $b);
        } else {
            printf("%-3d", $a * $b);
        }
    }
    printf("\n");
}

?>
